deployment/Dockerfile

```diff
@@ -33,6 +35,36 @@ RUN set -e -x -v; \
 	cd /root/llvm-project/build; \
 	make compiler-rt;
 
+
+# # Now build LibAFL
```

Comment:
I know we briefly discussed this offline, but why are we commenting out this section? Is this supposed to run conditionally? (If so, maybe move to a setup script or write conditional statements in Bash?)